ray-operator/controllers/ray/utils/dashboard_httpclient.go

```diff
@@ -197,10 +198,10 @@ type RayJobInfo struct {
 }
 
 // RayJobRequest is the request body to submit.
-// Reference to https://docs.ray.io/en/latest/cluster/jobs-package-ref.html#jobsubmissionclient.
+// Reference to https://docs.ray.io/en/latest/cluster/running-applications/job-submission/rest.html#ray-job-rest-api-spec
```

Comment:
Check the OpenAPI https://docs.ray.io/en/latest/cluster/running-applications/job-submission/api.html#/paths/~1api~1jobs/post for more details. Some types seem to be wrong.

Reply:
(And if we notice something is off, we can update the OpenAPI doc)